build/all/Dockerfile

```diff
@@ -171,6 +171,8 @@ RUN apt-get update && apt-get install -y bash git
 RUN mkdir -p /opt/nomos/bin
 WORKDIR /opt/nomos/bin
 COPY --from=bins /go/bin/nomos nomos
+COPY --from=bins /usr/local/bin/helm /usr/local/bin/helm
+COPY --from=bins /usr/local/bin/kustomize /usr/local/bin/kustomize
```

Comment:
Seems like a reasonable change

I think it would be worth adding some simple assertions (e.g. `helm`/`kustomize` version) to [this test case](https://github.com/GoogleContainerTools/kpt-config-sync/blob/main/e2e/testcases/cli_test.go#L925) so that we have a bit of a smoke test for these binaries in this image. Note that this image isn't used by any of the backend components so it needs targeted tests. I'd also be okay with adding some test cases as a followup PR

Reply:
Thanks for pointing out the test.  Adding additional checks there!  Thnx